pyproject.toml

```diff
@@ -55,7 +54,8 @@ keywords = [
 name = "cf-units"
 readme = "README.md"
 requires-python = ">=3.10"
-license.file = "LICENSE"
+license = "BSD-3-Clause"
```

Comment:
This caused me some concern since the core metadata 'Licence' field is deprecated.
But I finally spotted that the relationship between Core Metadata terms and pyproject.toml keys is non-trivial.
So, in this case, ["licence" --> "Licence Files"](https://packaging.python.org/en/latest/specifications/pyproject-toml/#license)
Likewise "license-files" --> "Licence File", and "file" -- "Licence" (deprecated).

Unexpectedly confusing!